src/Tizen.NUI/src/public/VisualMaps.cs

```diff
@@ -661,17 +661,21 @@ internal void UpdateVisual(bool force = false)
         /// <since_tizen> 3 </since_tizen>
         protected virtual void ComposingPropertyMap()
         {
-            if (null == _outputVisualMap)
-            {
-                _outputVisualMap = new PropertyMap();
-            }
+            _outputVisualMap = new PropertyMap();
```

Comment:
it looks like _outputVisualMap is created at every time. it seems different from the previous behavior. (previously, it is checked as null) how do think about? ^^;

Reply:
If the _outputVisualMap is not created everytime, the map can have multiple values for the same key (MIX_COLOR, OPACITY..)
Because `PropertyMap.Add(key, value)` does not check duplicates. (DALi says it's better to create new one instead reusing)

Reply:
OK. I see. thank you for explaining.